slides/kube/daemonset.md

```diff
@@ -202,6 +202,8 @@ And one too many pods...
 
   - *one pod per node* for the daemonset
 
+.footnote[(We don't run these pods on master nodes, so it's *one pod per worker node*.)]
```

Comment:
Do we (or the k8s docs) call them "worker nodes"? (Perhaps it's too early in the morning and I can't remember clearly...)

I agree that we should add a clarification but I'm going to hold off a bit for the exact wording.

Perhaps... `- except for the node hosting the control plane` (not crazy excited about this formulation, we'll see after a few days of sleep!)

Reply:
Aggravatingly but unsurprisingly, it appears the term "node" is semantically overburdened. 

Exhibit A: [A node is a worker machine in Kubernetes, previously known as a minion.](https://kubernetes.io/docs/concepts/architecture/nodes/)
Exhibit B: [The Kubernetes control plane is split into a set of components, which can all run on a single master node](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/architecture.md#cluster-control-plane-aka-master)